consensus/polybft/state_sync_manager.go

```diff
@@ -22,8 +22,10 @@ import (
 )
 
 const (
-	// number of stateSyncEvents to be processed before a commitment message can be created and gossiped
-	stateSyncCommitmentSize = 10
+	// maximum number of stateSyncEvents that a commitment can have
+	maxCommitmentSize = 10
```

Comment:
Can you make this value a param of the manager? This will likely not be a global variable in the future so it would be better if we do not forget about that (by not making it a global variable).

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1083/commits/5536eee6702ed7cf4159803120aa141c8d1bfce7